src/Main.cc

```diff
@@ -79,13 +79,21 @@ using sta::stringEq;
 using std::string;
 
 #ifdef ENABLE_PYTHON3
+#define PY_MODULES_WITHOUT_OPENROAD \
```

Comment:
This should take X as an arg

Reply:
eg https://github.com/The-OpenROAD-Project/OpenROAD/blob/846c27706460b3bbce5d584eaab8c2bec8a3d49b/src/utl/include/utl/Logger.h#L58

Reply:
Yes, that is less obscure. Will do.